apstra/api_iba_probes.go

```diff
@@ -0,0 +1,90 @@
+package apstra
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	apiUrlIbaProbes       = "/api/blueprints/%s/probes"
+	apiUrlIbaProbesPrefix = apiUrlIbaProbes + apiUrlPathDelim
+	apiUrlIbaProbesById   = apiUrlIbaProbesPrefix + "%s"
+)
+
+type IbaProbe struct {
```

Comment:
There's a couple of things that look like timestamps in here. Should they be `*time.Time`?

Reply:
I removed them since they don't matter for creating widgets and dashboards